src/main/io/osd.h

```diff
@@ -126,11 +126,15 @@ typedef enum {
     OSD_TIMER_PREC_COUNT
 } osd_timer_precision_e;
 
+#define OSD_PAGE_COUNT 3
+
 typedef enum {
-    // visible, bit 0
-    OSD_FLAG_VISIBLE = (1 << 0),
+    // visible, bits 0.2
+    OSD_FLAG_VISIBLE_PAGE_1 = (1 << 0),
+    OSD_FLAG_VISIBLE_PAGE_2 = (1 << 1),
+    OSD_FLAG_VISIBLE_PAGE_3 = (1 << 2),
```

Comment:
Indexing from 1 was done because this represents a tangible thing, rather than an array index. In the same way that the first motor is not motor 0.